lib/matplotlib/axes/_axes.py

```diff
@@ -1978,7 +1978,8 @@ def acorr(self, x, **kwargs):
 
         Parameters
         ----------
-        x : array-like
+        x : array-like.  x is not run through Matplotlib's unit
+            conversion, so this should be a unit-less array.
```

Comment:
```suggestion
        x : array-like.  *x* is not run through Matplotlib's unit
            conversion, so this should be a unit-less array.
```

Reply:
The description should be on a new line; right now half the description is part of the type.